packages/donor/src/components/Wizard/WizardScreen.tsx

```diff
@@ -1,40 +1,103 @@
-import Stepper from "../basic/Stepper";
-import Button, { ButtonVariant } from "../basic/Button";
+import Button from "../basic/Button/ButtonV2";
+import Text from "../basic/Text";
+import styles from "./WizardScreen.module.scss";
+import { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import MobileStepper from "@material-ui/core/MobileStepper";
+import { BrowserRouter, Link } from "react-router-dom";
+import { PropTypes } from "@material-ui/core";
+import { ButtonVariant } from "../basic/Button";
 
-export interface WizardScreenProps {
-  imageUrl: string;
-  title: string;
-  content: string;
+const useStyles = makeStyles({
```

Comment:
I don't think we should use makeStyles here, as we don't use it anywhere in the app.

Please see how we use styles in other components (by importing the css file to a variable) and do the same

Reply:
i couldn't make it work through the css file
i guess it had something to do with that those are inner mui mobilestepper elements